src/components/menu/menu.stories.mdx

```diff
@@ -333,13 +372,52 @@ is rendered.
   </Story>
 </Preview>
 
+## Dark icon
+
+Menus with icons only must have a supplied keyboard shortcut. If no shortcut is supplied the menu item text is navigable by the first letter of each item. 
+The menu must have focus for this to work
+
+<Preview>
+  <Story name="dark icon" parameters={{ chromatic: { disable: true } }}>
```

Comment:
I would not ignore this story in `chromatic` because we have shown that we can actually use the `Icon` component in `Menu`

Reply:
The default icon story is in chromatic already and we have other dark theme stories being recorded. If you still think we need this we can enable it.